static/templates/keyboard_shortcuts.hbs

```diff
@@ -193,45 +193,45 @@
             <table class="hotkeys_table table table-striped table-bordered table-condensed">
                 <thead>
                     <tr>
-                        <th colspan="2">{{ _("Message actions") }}</th>
+                        <th colspan="2">{{t 'Message actions' }}</th>
                     </tr>
                 </thead>
                 <tr>
-                    <td class="definition">{% trans %}Edit your last message{% endtrans %}</td>
+                    <td class="definition">{{t 'Edit your last message' }}</td>
                     <td><span class="hotkey"><kbd class="arrow-key">←</kbd></span></td>
                 </tr>
                 <tr>
-                    <td class="definition">{% trans %}Show message sender's profile{% endtrans %}</td>
+                    <td class="definition">{{t "Show message sender's profile"   }}</td>
                     <td><span class="hotkey"><kbd>U</kbd></span></td>
                 </tr>
                 <tr>
-                    <td class="definition">{% trans %}Show images in thread{% endtrans %}</td>
+                    <td class="definition">{{t 'Show images in thread' }}</td>
                     <td><span class="hotkey"><kbd>V</kbd></span></td>
                 </tr>
                 <tr id="edit-message-hotkey-help">
-                    <td class="definition">{% trans %}Edit selected message{% endtrans %}</td>
+                    <td class="definition">{{t 'Edit selected message' }}</td>
                     <td><span class="hotkey"><kbd>E</kbd></span></td>
                 </tr>
                 <tr>
-                    <td class="definition">{% trans %}Star selected message{% endtrans %}</td>
+                    <td class="definition">{{t 'Star selected message' }}</td>
                     <td><span class="hotkey"><kbd>Ctrl</kbd> + <kbd>S</kbd></span></td>
                 </tr>
                 <tr>
                     <td class="definition">
-                        {% trans %}React to selected message with{% endtrans %}
-                        <img alt=":thumbs_up:"
+                        {{t 'React to selected message with' }}
+                        <img alt="{{t ':thumbs_up:' }}"
```

Comment:
This string was intentionally not tagged for translation, because it's an emoji reference, aka a unique identifier.  Adding this back, as well as its lint error.